packages/venia-concept/package.json

```diff
@@ -18,15 +18,13 @@
     "build:report": "buildpack generate-build-report",
     "buildpack": "buildpack",
     "clean": "rimraf dist",
-    "download-schema": "graphql get-schema --project venia --insecure",
     "lint": "eslint --ignore-path .gitignore 'src/**/{*.js,package.json}'",
     "prettier": "prettier --loglevel warn --ignore-path .gitignore 'src/**/*.@(css|graphql|js)' '*.js'",
     "prettier:check": "yarn run -s prettier -- --check",
     "prettier:fix": "yarn run -s prettier -- --write",
     "start": "buildpack serve .",
     "start:debug": "node --inspect-brk ./node_modules/.bin/webpack-dev-server --progress --color --env.mode development",
     "test": "yarn run -s prettier:check && yarn run -s lint && jest",
-    "validate-queries": "yarn run download-schema && graphql validate-magento-pwa-queries --project venia",
```

Comment:
`yarn build:dev` uses this command still - will removing it cause breaks?

Also how will folks validate their queries in their scaffolded apps without this script/

Reply:
This is the fileglob that was previously in `.graphqlconfig` that was used by the plugin. `validate-queries` didn't actually do anything in scaffolded apps 😬  those packages would be in node_modules on a scaffold. The feature parity I was aiming for was just to restore validation in the monorepo.

`"filesGlob": "../{peregrine,venia-ui,venia-concept}/{lib,src}/**/*.{js,graphql,gql}"`

Reply:
Removed from `build:dev` in b9de346

Reply:
> validate-queries didn't actually do anything in scaffolded apps

I am not 100% sure that `validate-queries` is not doing nothing on scaffolded apps. You can test this by pointing your MAGENTO_BACKEND_URL at an invalid server. If you then run yarn watch prior to this PR you will get something like

```
  message: 'invalid json response body at https://magento2.foomanstore.test/graphql reason: Unexpected token p in JSON at position 4',
  type: 'invalid-json'
```

So while we may not validate any queries at least grapqhl schema needs to generate successfully (funnily we have had a case where the back-end schema did not work but all other queries did).

Reply:
@fooman - This was a tiny reason why I left the `graphql-config@2` or `.graphqlconfig` around in `venia-concept`, but for a different reason; so IDEs would still be able to fetch schemas to provide in editor validation (eg. I use [this webstorm plugin](https://github.com/jimkyndemeyer/js-graphql-intellij-plugin), which doesn't yet support v3).

Ideal state is that validation works in scaffolded apps the same as the monorepo, and the state in this PR is no validation. We do introspection at build time for fragment matching, so I'm not sure a `validate-queries` command would indicate much, but happy to hear any ideas for a middle ground state that isn't just a removal of this command. Is this more about the nice messaging the previous plugin provided?

Reply:
Thanks for getting back to me. Can confirm that latest scaffolded is not working well with `validate-queries` (it may have done so in the past as did see a `lastCachedGraphQLSchema.json` at some point). So I guess the suggestion here is reasonable to fix it first for the monorepo and then hopefully we can get this to work with scaffolded apps as the next step.

Reply:
With quite a bit of tinkering I am now able to run validate-queries locally in a scaffolded apps.

In rough steps (I am not running the same apollo version which may be the reason for step 1):
1. force a graphql resolution
2. edit .graphqlconfig to find the files under node_modules with `"filesGlob": "./node_modules/@magento/{peregrine,venia-ui,venia-concept}/{lib,src}/**/*.{graphql,gql,gql.js}",`
3. overcome default node_module exclusion by adding `ignorePattern: '!node_modules/*'` to linterConfiguration
to node_modules/graphql-cli-validate-magento-pwa-queries/lib/index.js
4.  Profit? https://github.com/magento/pwa-studio/issues/3217

Reply:
@fooman 
Given how tech debt is usually treated I'll see if I can make that happen in this scope. PR welcome, otherwise I'll work on it tomorrow.